Detector/DetFCChhTrackerTkLayout/src/BeamTube_geo.cpp

```diff
@@ -0,0 +1,35 @@
+
```

Comment:
Seems like a very good idea. Currently our beam-pipe uses the generic Cylinder factory 
https://github.com/HEP-FCC/FCCSW/blob/master/Detector/DetFCChhBaseline1/compact/FCChh_BeamTube.xml#L20
I don't think it's good to add extensions specific to the beam pipe there. Should I move this factory instead to DetFCChhBaseline1?